resources/main.css

```diff
@@ -1,3 +1,13 @@
+a {
+    color: white;
+    font-family: Roboto,sans-serif;
+}
+
+button {
```

Comment:
So, you're saying all buttons on all pages should always have a bottom margin of 30pt?